README.md

```diff
@@ -123,6 +123,17 @@ Unstable macOS installers are generated after every commit to the master branch,
 
 - [Download Installer](https://github.com/Nicotine-Plus/nicotine-plus/actions?query=branch%3Amaster+event%3Apush+is%3Asuccess+workflow%3A%22Packaging%22)
 
+## OpenBSD
```

Comment:
This should be under the `GNU/Linux, *BSD` section, before `Other Distributions`.

Reply:
Ping?